brigade/core/configuration.py

```diff
@@ -0,0 +1,61 @@
+import ast
+import os
+
+
+import yaml
+
+
+CONF = {
```

Comment:
I'd suggest using `marshmallow` or somehting like that:

https://marshmallow.readthedocs.io/en/latest/

Reply:
maybe it's too heavy...

Reply:
I think it would be a shame to add a requirement just for this. We don't need any advanced features. It will only be a matter of inputting data in that section and have some tests to verify.

Reply:
Also if desired that part is really easy to change in the future. Might depend on how many parameters that will need to be defined.